integration-cli/docker_cli_rename_test.go

```diff
@@ -39,8 +45,7 @@ func (s *DockerSuite) TestRenameRunningContainer(c *check.C) {
 }
 
 func (s *DockerSuite) TestRenameRunningContainerAndReuse(c *check.C) {
-	testRequires(c, DaemonIsLinux)
-	out, _ := dockerCmd(c, "run", "--name", "first_name", "-d", "busybox", "top")
+	out, _ := dockerCmd(c, "run", "--name", "first_name", "-d", "busybox", sleepCmd, "60")
```

Comment:
top doesn't exist in Windows busybox (or msys2). I have to have something that will reliably keep a container running, sleep 60 works. I've run Linux CI multiple times with these changes and not seen any flakiness.